postgres/content.md

```diff
@@ -40,6 +40,29 @@ postgres=# SELECT 1;
 
 Run `docker stack deploy -c stack.yml %%REPO%%` (or `docker-compose -f stack.yml up`), wait for it to initialize completely, and visit `http://swarm-ip:8080`, `http://localhost:8080`, or `http://host-ip:8080` (as appropriate).
 
+## Healtcheck
+
+To add a health check to the image use the following command:
+
+```console
+docker run --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword \
```

Comment:
Do you need `--name some-postgres`? I feel like for demonstration purposes, it's not necessary

Reply:
This is to stay consistent with the previous example in the docs

Reply:
https://github.com/docker-library/docs/blob/master/postgres/content.md?plain=1#L16

Reply:
Okay gotchu!  Consistency is good.

Thanks for answering, you have my approval 👍 🚀